arch/x86/core/ia32/excstub.S

```diff
@@ -164,6 +164,7 @@ SECTION_FUNC(PINNED_TEXT, _exception_enter)
 	testl	$0x200, __z_arch_esf_t_eflags_OFFSET(%esp)
 	je	allDone
 	sti
+	nop
```

Comment:
This one is safe as-is, since the stack pointer won't (must not!) change across an interrupt, if it did then we have a critical bug in our context restore.